app/services/clients/connector.client.js

```diff
@@ -516,181 +427,120 @@ ConnectorClient.prototype = {
-        service: SERVICE_NAME
+        op: 'replace',
+        path: stripeAccountSetupFlag,
+        value: true
       }
-    )
+    ]
+    configureClient(client, url)
+    const response = await client.patch(url, body, 'set stripe account setup flag to true for gateway account')
+    return response.data
   },
 
-  getStripeAccount: function (gatewayAccountId) {
-    return baseClient.get(
-      {
-        baseUrl: this.connectorUrl,
-        url: STRIPE_ACCOUNT_PATH.replace('{accountId}', gatewayAccountId),
-        json: true,
-        description: 'get stripe account for gateway account',
-        service: SERVICE_NAME,
-        transform: responseBodyToStripeAccountTransformer
-      }
-    )
+  getStripeAccount: async function (gatewayAccountId) {
+    const url = `${this.connectorUrl}${STRIPE_ACCOUNT_PATH.replace('{accountId}', gatewayAccountId)}`
+    configureClient(client, url)
+    const response = await client.get(url, 'get stripe account for gateway account')
+    return responseBodyToStripeAccountTransformer(response.data)
   },
 
-  postChargeRequest: function (gatewayAccountId, payload) {
-    return baseClient.post(
-      {
-        baseUrl: this.connectorUrl,
-        url: CHARGES_API_PATH.replace('{accountId}', gatewayAccountId),
-        json: true,
-        body: payload,
-        description: 'create payment',
-        service: SERVICE_NAME
-      }
-    )
+  postChargeRequest: async function (gatewayAccountId, payload) {
+    const url = `${this.connectorUrl}${CHARGES_API_PATH.replace('{accountId}', gatewayAccountId)}`
+    configureClient(client, url)
+    const response = await client.post(url, payload, 'create payment')
+    return response.data
   },
 
-  getCharge: function (gatewayAccountId, chargeExternalId) {
-    const url = CHARGE_API_PATH.replace('{accountId}', gatewayAccountId).replace('{chargeId}', chargeExternalId)
-    return baseClient.get({
-      baseUrl: this.connectorUrl,
-      url,
-      description: 'get a charge',
-      service: SERVICE_NAME
-    })
+  getCharge: async function (gatewayAccountId, chargeExternalId) {
+    const url = `${this.connectorUrl}${CHARGE_API_PATH.replace('{accountId}', gatewayAccountId).replace('{chargeId}', chargeExternalId)}`
+    configureClient(client, url)
+    const response = await client.get(url, 'get a charge')
+    return response.data
   },
 
-  postAccountSwitchPSP: function (gatewayAccountId, payload) {
-    const url = SWITCH_PSP_PATH.replace('{accountId}', gatewayAccountId)
-    return baseClient.post({
-      baseUrl: this.connectorUrl,
-      url,
-      body: payload,
-      description: 'switch account payment service provider',
-      service: SERVICE_NAME
-    })
+  postAccountSwitchPSP: async function (gatewayAccountId, payload) {
+    const url = `${this.connectorUrl}${SWITCH_PSP_PATH.replace('{accountId}', gatewayAccountId)}`
+    configureClient(client, url)
+    const response = await client.post(url, payload, 'get a charge')
```

Comment:
Description needs to be updated to match

Reply:
Changed as requested. Also spotted another i.e. `updateConfirmationEmailEnabled` description s/b `'update confirmation email enabled'`